backend/CircleForms/Services/Database/UserRepository.cs

```diff
@@ -8,31 +7,40 @@
 
 namespace CircleForms.Services.Database
 {
-    public class UserDatabaseService : IUserDatabaseService
+    public class UserRepository : IUserRepository
     {
         private readonly IMongoCollection<User> _users;
 
-        public UserDatabaseService(IConfiguration config)
+        public UserRepository(IConfiguration config)
         {
             var client = new MongoClient(config.GetConnectionString("Database"));
             var database = client.GetDatabase("circleforms");
 
-            _users = database.GetCollection<User>("Users");
+            _users = database.GetCollection<User>("users");
         }
 
         public async Task<List<User>> Get()
         {
-            return await _users.Find(x => true).ToListAsync();
+            return await (await _users.FindAsync(x => true)).ToListAsync();
```

Comment:
Unless you want to use the cursor `FindAsync` here is not required. `Find(...).ToListAsync()` will yield the same result
https://stackoverflow.com/questions/30650722/difference-between-find-and-findasync

Reply:
`FindAsync `is just more optimized to retrieve specific data asynchronously than `Find().ToListAsync()` so there's no reason to revert

Reply:
We're returning everything here though. It would make more sense if we were iterating through returned data, but we're not.

Reply:
oh yeah it's kinda true 🤒